actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -74,10 +82,12 @@ class ActionCable.Connection
 
   events:
     message: (event) ->
+      return if @disconnected
```

Comment:
Yes we could. I ran through some scenarios where we could just tell the server to stop communicating in these cases, but it gets kind of murky and the websocket protocol seems to indicate the onus is on the client to disconnect when it doesn't like the protocol not the server. Also if somebody writes a client in something other than in the browser (that would be me :-P), they might not want that behavior.